contrib/platform/test/com/sun/jna/platform/win32/Crypt32Test.java

```diff
@@ -468,7 +468,6 @@ private void enumerateRootCertificates(HCERTSTORE hCertStore) {
                 assertNotNull(ctx.pCertInfo.Subject);
                 assertFalse(decodeName(ctx.pCertInfo.Issuer).isEmpty());
                 assertFalse(decodeName(ctx.pCertInfo.Subject).isEmpty());
-                assertEquals(decodeName(ctx.pCertInfo.Issuer), decodeName(ctx.pCertInfo.Subject));
```

Comment:
Why?

Reply:
Because the assumption does not necessarily hold.:

![image](https://github.com/java-native-access/jna/assets/2179736/bb9bfc04-83a4-4a55-8bd5-b4d41902e1bd)

I guess, that these certificates are installed by Oracle VirtualBox Guest Extensions.